py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -105,22 +105,6 @@ def alias_replacement(x: torch.Tensor) -> torch.Tensor:
     return x
 
 
-@register_torch_trt_decomposition(
-    torch.ops.aten.addmm, registry=TORCH_TRT_DECOMPOSITIONS
-)
-def addmm_replacement(
-    input_: torch.Tensor,
-    mat1: torch.Tensor,
-    mat2: torch.Tensor,
-    *,
-    beta: int = 1,
-    alpha: int = 1,
-) -> torch.Tensor:
-    return torch.add(
-        torch.mul(input_, beta), torch.mul(torch.matmul(mat1, mat2), alpha)
-    )
```

Comment:
This PR also requires a converter for `aten.addmm`, since we are no longer decomposing it, so it would be a decrease in coverage otherwise

Reply:
Additionally, you can remove the `addmm` lowering pass tests, since the functionality is no longer present